### PR TITLE
Started writing readme for proper info about the project

### PR DESCRIPTION
**Pull request by him1411**

### CODE REVIEW

**Comment by him1411** [comment]
updated a typo

**Comment by him1411** [comment]
updated a typo

File: README.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
-# Fragments-Download
-This script downloads anyurl by fragmenting the file and increases speed drastically.
+# All purpose downloader 
+## Powered by CRUx - The computing and programming club 
 
-usage: python3 frag.py URL
+### BITS Pilani Hyderabad Campus
```
Comment (aero31aero) [inline README.md]: Unnecessary use of headings. Plus, no need to specify college name at the top.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
-# Fragments-Download
-This script downloads anyurl by fragmenting the file and increases speed drastically.
+# All purpose downloader 
+## Powered by CRUx - The computing and programming club 
 
-usage: python3 frag.py URL
```
Comment (aero31aero) [inline README.md]: Add usage in its own section: Dependency installation etc? Also, use single tildes for writing code snippets like: `python3 ...`

File: README.md [hunk 3]
```diff
@@ -1,5 +1,13 @@
-# Fragments-Download
-This script downloads anyurl by fragmenting the file and increases speed drastically.
+# All purpose downloader 
+## Powered by CRUx - The computing and programming club
```
Comment (aero31aero) [inline README.md]: Powered by sounds too extreme. Again, unnecessary use of heading.

File: README.md [hunk 4]
```diff
@@ -1,5 +1,13 @@
-# Fragments-Download
-This script downloads anyurl by fragmenting the file and increases speed drastically.
+# All purpose downloader 
+## Powered by CRUx - The computing and programming club 
 
-usage: python3 frag.py URL
+### BITS Pilani Hyderabad Campus
+![](https://scontent.fbom1-1.fna.fbcdn.net/v/t1.0-1/p200x200/17904452_1491990900819437_3846562565023385004_n.png?oh=4a26af21c3764e36319197a532de47c7&oe=59DFB8EC)
```
Comment (aero31aero) [inline README.md]: Can this be changed to a better hosted logo? If you have free time, create a PR to add this into the `cruxbphc.github.io` repo and link from there.

**Review by aero31aero — CHANGES_REQUESTED** [review]
Needs Contributing guide as well as a License. Good draft, keep it up. :)

File: README.md (hunk 3 above)
Comment (him1411) [reply README.md]: will do

File: README.md (hunk 1 above)
Comment (him1411) [reply README.md]: I think college name is kind of essential because the club is because of the college. Just an opinion

File: README.md (hunk 2 above)
Comment (him1411) [reply README.md]: Writing all that now. Will finish it by tonight

File: README.md (hunk 1 above)
Comment (aero31aero) [reply README.md]: But someone happening to come to the README file probably doesn't care about who made this and where but rather, what it does. If they like it, then they'd look at other things. Again, its your project so you could take any route, but that's how I've seen it done.

**Comment by him1411** [comment]
will do that by tonight . Thanks

File: README.md (hunk 1 above)
Comment (him1411) [reply README.md]: should I write these things at the bottom then ? will be more suitable I suppose ?

**Comment by aero31aero** [comment]
Also, closing an issue/PR means its done. You only close them when:
1. The commits have been merged OR issue has been resolved.
2. The commits have been rejected OR issue has been marked as `wont-fix`.

**Comment by him1411** [comment]
Didn't know that.Thanks

**Comment by him1411** [comment]
Made the changes proposed.

**Pull request merged**